### ISSUE TITLE
variables shared by groups

### ISSUE THREAD

**Issue by Natkeeran** [Islandora/documentation#1434]
What is the best practice to define variables such as solr_cores needed by multiple groups?  For instance, solr_cores is needed by solr to setup the core and webserver to configure the drupal module.  

Can we define it here https://github.com/Islandora-Devops/islandora-playbook/blob/dev/inventory/vagrant/group_vars/all/main.yml ?

Shared Variable List
* solr_cores:
* apache_listen_port: 8000

**Comment by dannylamb** [comment]
Yes.  Any file in the `all` directory is applied to all ansible groups.
